packages/web-awesome/src/components/app/TestResult/TestResultHistory/TestResultHistoryItem.tsx

```diff
@@ -9,38 +9,49 @@ import TreeItemIcon from "@/components/app/Tree/TreeItemIcon";
 import { IconButton } from "@/components/commons/Button";
 import { TooltipWrapper } from "@/components/commons/Tooltip";
 import { Text } from "@/components/commons/Typography";
-import { navigateTo } from "@/index";
+import { openInNewTab } from "@/index";
+import { useI18n } from "@/stores";
 import { timestampToDate } from "@/utils/time";
 
 export const TestResultHistoryItem = ({ testResultItem }) => {
-  const { status, message, trace, stop, duration, id, uuid } = testResultItem;
+  const { status, message, trace, stop, duration, id } = testResultItem;
   const [isOpened, setIsOpen] = useState(false);
   const convertedStop = timestampToDate(stop);
-  const formattedDuration = formatDuration(duration);
+  const formattedDuration = formatDuration(duration as number);
+  const { t } = useI18n("controls");
 
-  const navigateUrl = `/${uuid}/${id}`;
+  const navigateUrl = `/testresult/${id}`;
 
   return (
     <div>
-      <div className={styles["test-result-history-item-header"]} onClick={() => setIsOpen(!isOpened)}>
-        {Boolean(message) && <ArrowButton isOpened={isOpened} icon={arrowsChevronDown.id} />}
-        <div className={styles["test-result-history-item-wrap"]}>
+      <div className={styles["test-result-history-item-header"]}>
+        {Boolean(message) && (
+          <span onClick={() => setIsOpen(!isOpened)}>
+            <ArrowButton isOpened={isOpened} icon={arrowsChevronDown.id} />
+          </span>
+        )}
+        <div
+          className={styles["test-result-history-item-wrap"]}
+          onClick={(e) => {
+            e.stopPropagation();
+            openInNewTab(navigateUrl);
```

Comment:
Let's open the history entry in the same tab when the row has been clicked. In current implementation, it doesn't make sense to have `Open in new tab` button when the row works in the same way